addons/pinniped/config-controller/controllers/handlers.go

```diff
@@ -20,8 +20,9 @@ import (
 
 func (c *PinnipedController) configMapToCluster(o client.Object) []ctrl.Request {
 	// return empty object, if pinniped-info CM changes, update all the secrets
-	c.Log.V(1).Info("configmap created/updated/deleted, sending back empty request to reconcile all clusters")
-	return []ctrl.Request{}
+	return []ctrl.Request{{
+		NamespacedName: client.ObjectKey{Namespace: o.GetNamespace(), Name: o.GetName()},
```

Comment:
i was going to say we could list all of the clusters in `configMapToCluster()` and return them as `ctrl.Request`'s, i.e. (pseudocode):
```golang
func (c *PinnipedController) configMapToCluster(o client.Object) []ctrl.Request {
  reqs := []ctrl.Request{}
  clusters := c.client.ListClusters(context.WithTimeout(time.Second*3))
  for _, cluster := range clusters {
    reqs = append(reqs, ctrl.Request{cluster})
  }
  return reqs
}
```

but that seems like a bit of an anti pattern because a `context.Context` isn't passed through to the map function? i was going to go read other controllers and see what they do, but didn't want to spend anymore time on it. i think we are a bit special because we are using a singleton as our parent object (`ConfigMap`). so i dunno what to do there. the next steps i would take is to go read some other controller-runtime controllers that sync singleton objects.

Reply:
ok, so I am a fool, this will work if we pass in `[]ctrl.Request{{}}` vs `[]ctrl.Request{}`... this ends up sending the following request object: `{"Namespace":"","Name":""}`.  lmk what you think about that.  I'm still checking `req == ctrl.Request{}` in the Reconcile function, but we could check namespace/name =="" instead.  Not sure which is better.